src/indigo/less/new-ui.less

```diff
@@ -0,0 +1,266 @@
+@primary: #228dff;
```

Comment:
@natocTo ak by ti to nevadilo, skusme tieto premenne nejako oprefixovat, aby sa nam nestalo, ze prepiseme nejaku existujucu (@brand-primary alebo tak). Kludne ako `new-ui-primary` alebo inak ako uznas za vhodne.

Reply:
Přidám tam `new-ui`.